opentile/common.py

```diff
@@ -346,10 +345,10 @@ def get_tile(
         """
         tile_point = Point.from_tuple(tile_position)
         frame_index = self._tile_point_to_frame_index(tile_point)
-        frame = self._read_frame(frame_index)
-        if self.page.jpegtables is None:
-            return frame
-        return Jpeg.add_jpeg_tables(frame, self.page.jpegtables)
+        tile = self._read_frame(frame_index)
```

Comment:
Can tile be None here? Then return type needs to be optional. And same as above, but the main path without if-clauses and special cases in the if-clause. So from what I can see the, original was good here?

Reply:
Tile can't be None. This arrangement fits better with how add_jpeg_tables() is used elsewhere, where more operations might be run on the tile.